libs/core/build.gradle

```diff
@@ -22,6 +22,7 @@ apply plugin: 'nebula.maven-base-publish'
 apply plugin: 'nebula.maven-scm'
 
 dependencies {
+    compileOnly "com.google.code.findbugs:jsr305:3.0.2"
```

Comment:
We should add a note here explaining the purpose of this dependency. Also so folks don't accidentally copy this when creating new `:libs` subprojects. Let's add a comment: `// This dependency is used only by :libs:core for null-checking interop with other tools`

Reply:
Done